temba/contacts/views.py

```diff
@@ -1081,6 +1081,21 @@ def get_context_data(self, *args, **kwargs):
             context["reply_disabled"] = True
             return context
 
+        def get_gear_links(self):
+            links = []
+
+            if self.has_org_perm("contacts.contact_export"):
```

Comment:
Hote: Not a big deal, but probaby it will be good to move that piece of code to the parent class `ContactListView`

Reply:
Yep, let's make it in another ticket. Good catch!